analysis/import/SonarImporter/src/main/kotlin/de/maibornwolff/codecharta/importer/sonar/dataaccess/SonarMeasuresAPIDatasource.kt

```diff
@@ -51,44 +51,52 @@ class SonarMeasuresAPIDatasource(private val user: String, private val baseUrl:
     private val client: Client = ClientBuilder.newClient()
     private val logger = KotlinLogging.logger {}
 
+    private var measureBatches = 0
+    private var processedPages = 0
+
     init {
         client.register(ErrorResponseFilter::class.java)
         client.register(GsonProvider::class.java)
     }
 
     fun getComponentMap(componentKey: String, metricsList: List<String>): ComponentMap {
```

Comment:
And what is a component? Is it one file in the sonar-world?

Reply:
essentially yes; but it contains only some and not necessarily all measures/metrics of a file